src/integrant/repl.clj

```diff
@@ -44,18 +46,22 @@
              {:resume-exception %1}
              %2)))
 
-(defn init []
-  (alter-var-root #'state/system (fn [sys]
-                                   (halt-system sys)
-                                   (init-system state/config)))
-  :initiated)
+(defn init
+  ([] (init nil))
+  ([keys]
+   (alter-var-root #'state/system (fn [sys]
+                                    (halt-system sys)
+                                    (init-system state/config keys)))
+   :initiated))
 
-(defn go []
-  (prep)
-  (init))
+(defn go
+  ([] (go nil))
+  ([keys]
+   (prep)
+   (init keys)))
 
 (defn clear []
-  (alter-var-root #'state/system (fn [sys] (halt-system sys) nil))
```

Comment:
This looks like a typo

Reply:
Yes, I think this change is probably a mistake.

Reply:
Yup, no idea how that happened. I'll fix it soon.

Reply:
Weird, my local copy doesn't even show this line changing or being deleted, and there's no changes visible in git... I'm really not sure what happened or even how to fix it. Maybe I'll try squashing

Reply:
Even after rebasing on master and squashing everything into one commit, this line still appears deleted on GitHub, but not on my local machine.

Reply:
Could you pull my branch and confirm?

Reply:
Er, wait, sorry, I'm just blind. I thought the line was deleted but it was just a paren that was moved.